Resources/keybinds.yml

```diff
@@ -258,6 +258,9 @@ binds:
 - function: OfferItem
   type: State
   key: F
+- function: ToggleStanding
+  type: State
+  key: R
```

Comment:
*dies of cringe*

Reply:
wait no that's how 13 does it; still wack.

Reply:
R is an odd default. *Many of the defaults are...*

Reply:
R is short for "rest". I thought of L and similar keys, but they're all occupied.